### PR TITLE
notify on study creation error

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/components/study-manager.js [hunk 1]
```diff
@@ -494,7 +499,25 @@ const StudyManager = ({ onClick }) => {
         const ws = connectNotificationsWsUpdateStudies();
 
         ws.onmessage = function (event) {
-            dispatchStudies();
+            let eventData = JSON.parse(event.data);
+            if (eventData.headers) {
```
Comment (SlimaneAmar) [inline src/components/study-manager.js:503]: Perhaps a function `displayErrorIfExist`

File: src/components/study-manager.js [hunk 2]
```diff
@@ -494,7 +499,25 @@ const StudyManager = ({ onClick }) => {
...
+                const error = eventData.headers['error'];
+                if (error !== undefined) {
+                    const studyName = eventData.headers['studyName'];
+                    const errorMessage =
```
Comment (SlimaneAmar) [inline src/components/study-manager.js:507]: Perhaps a const for hearder name `error` or const enum for all header names of te web socket (study)

File: src/components/study-manager.js [hunk 3]
```diff
@@ -480,6 +483,8 @@ const StudyManager = ({ onClick }) => {
 
     const classes = useStyles();
 
+    const { enqueueSnackbar } = useSnackbar();
+
```
Comment (SlimaneAmar) [inline src/components/study-manager.js:487]: Why we not use commons-ui snackbar for this ?

File: src/components/study-manager.js (hunk 1 above)
Comment (FranckLecuyer) [reply src/components/study-manager.js:501]: Done

File: src/components/study-manager.js (hunk 3 above)
Comment (FranckLecuyer) [reply src/components/study-manager.js:487]: I'm using snackbar from commons-ui : see app.js where SnackbarProvider is imported and used

See demo/src/app.js in commons-ui, where there is an exemple of snackbar use

Comment (SlimaneAmar) [reply src/components/study-manager.js:487]: ok

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**